### PR TITLE
Developing

### PR DESCRIPTION
**Pull request by madcido**

### CODE REVIEW

File: css/stylesheet.css [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+    height: 80%;
+}
+#main{
+    width: 90%;
+    margin: 60px auto 0 auto;
+    display: grid;
+    grid-template-rows: 15vw auto 1fr;
+    grid-template-columns: repeat(3, 27vw);
+    grid-template-areas: 
+    "video video ads"
```
Comment (JamesOkunlade) [inline css/stylesheet.css]: Remove these lines.

**Review by JamesOkunlade — CHANGES_REQUESTED** [review]
Not a bad project for embedding videos, however, I think you can do a lot better. Think of the project as a portfolio you'll have to show off to potential employers. So I think you might want to work on the styling a bit more. :slightly_smiling_face:

If you're new to HTML & CSS, you need not to rush in doing your projects. What matters is you really learning before moving on to the next topic. If you're having problem making it look a little more like a real youtube page, fork a repository of someone in your standup that did a better work and study their code.

Reduce your usage of "id" and use "class" instead. Except when "id" is really needed.

1. There is a common error in your HTML img tags. Ensure it always contain the "alt" attribute.
2. Remove the frameborder attribute in your iframe tag and specify the style in your CSS instead.

Let's see how much you can improve it by working on it for a day or two. :sunglasses:
When you're done, request for another code review and add me as the TSE so I can follow up.

Happy coding!

**Comment by madcido** [comment]
Thanks for the feedback, this is pretty much my first project ever, as I started to learn HTML/CSS because of Microverse one month ago. About the styling, I didn't think of it as my portfolio at first and the odin project specification didn't mention any styling, so I thought it was ok. But now I see you're right, so I made it way more beautiful this time. I hope you enjoy it.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,607 @@
...
+                    <a href="https://www.youtube.com/" target="_blank">
+                        <img class="contained-img" src="images/800px-Logo_YouTube_por_Hernando.svg.png" alt="logo">
+                    </a>
+                </div>
+            </div>
+            <!--search bar-->
+            <div class="flex-row">
+                <input id="search-bar" type="search" placeholder="Search">
+                <button id="search-button" class="weak" type="submit" value="Submit">
+                    <div class="button auto-margin">
```
Comment (JamesOkunlade) [inline index.html]: You cannot use div as a child element of button. Remove the div tag and give the div styling to the button tag.

**Review by JamesOkunlade — APPROVED** [review]
Awesome job!! 

This is a superb improvement on your previous submission. Very impressive! Just one thing you should do now. 

Refactor line 28 of your HTML code. You cannot use a div as a child element of button.

When you're done, merge developing to master, close the pull request and fill the project completion form.

Here are your thumbs up :+1::+1:

Goodluck in future projects.

**Pull request merged**